### ISSUE TITLE
[HOLD for payment 2024-08-07] [Workspace Feeds] Design adjustments part 1

### ISSUE THREAD

**Issue by mountiny** [Expensify/App#45733]
Coming from some initial testing [here](https://expensify.slack.com/archives/C036QM0SLJK/p1721221346790209) the design team suggested some adjustments. @shawnborton @allgandalf @DylanDylann @VickyStash @narefyev91 

I believe @koko57 is fixing this one in https://github.com/Expensify/App/pull/45690

The empty state doesn't have the correct amount of padding on desktop... it should use 32px on desktop and 20px on mobile. Compare with the workspace empty state for example:
![image](https://github.com/user-attachments/assets/21e315f2-8ff0-4ed9-96fb-267c88f3fb59)
![image](https://github.com/user-attachments/assets/42937f97-9efd-44a1-882f-70e367c04d0a)

____

When going through the issue card flow, the selected limit type in Step 3 does not have the correct background color applied to it. Compare it to other list options for example:
![image](https://github.com/user-attachments/assets/98b7a449-c385-4f76-8a40-9977e2fde9d3)
![image](https://github.com/user-attachments/assets/76d3a3ff-546c-42cb-905d-f6b51c57df74)

____

The text below the headline of step 6 is too small (it's 13px, but it should be 15px) and is too far away from the headline (should be just 12px margin between headline and text from what I see elsewhere). Compare this to other stepped flows for example:
![image](https://github.com/user-attachments/assets/3020a5bc-c226-4584-8134-cc0507da3ac8)
![image](https://github.com/user-attachments/assets/3a4a924b-4278-4c38-a3f8-0c91d71f1b86)

____

Update the spacing at the empty state https://github.com/Expensify/App/pull/45690#issuecomment-2243156435

Let's add 32px of space below the two buttons and above the empty state rows.

![image](https://github.com/user-attachments/assets/5498811b-d5b4-4211-8109-dba7579428c6)

<details><summary>Issue Owner</summary>Current Issue Owner: @strepanier03</details>

**Comment by mountiny** [comment]
we will handle this internally

**Comment by koko57** [comment]
@mountiny please assign me - I will either create a separate PR for that or will do it together with the flow integration with the BE. The first problem with the padding is solved in https://github.com/Expensify/App/pull/45690

**Comment by trjExpensify** [comment]
Assigned!

**Comment by koko57** [comment]
I fixed the issues and I could open the PR now, but there is an improvement with option focus change implemented by @VickyStash here: https://github.com/Expensify/App/pull/45601 so it's better to wait on this one to be merged so these changes would also be incorporated here

**Comment by mountiny** [comment]
going to be handled soon

**Comment by mountiny** [comment]
There is one more little cleanup to add from @shawnborton https://github.com/Expensify/App/pull/45690#issuecomment-2243156435 adding it to the gh body too

**Comment by koko57** [comment]
not sure what happened here - while working on empty state modal PR this was looking ok on android - but now the SVG is cut off 
![Screenshot_1721738016](https://github.com/user-attachments/assets/4eec6abc-a1d7-4eb3-8fa6-77d597f8a16a)

I'm trying to fix it now, that why I'm still not opening the draft for review

**Comment by mountiny** [comment]
Thanks for an update

**Comment by koko57** [comment]
ok, ready for review

**Comment by strepanier03** [comment]
@mountiny - Do we need the checklist [here](https://github.com/Expensify/App/issues/45733#issuecomment-2261272779)? 

If not, feel free to close this, there is no payment due according to Melvin-bot.

**Comment by mountiny** [comment]
Yah I dont thinkwe need that here
